package.json

```diff
@@ -116,8 +127,10 @@
 		"babel-plugin-transform-vue-jsx": "^3.5.1",
 		"babel-preset-env": "^1.6.1",
 		"babel-preset-stage-2": "^6.22.0",
+		"babel-preset-vue": "^2.0.2",
 		"chalk": "^2.4.2",
 		"chokidar": "^3.1.0",
+		"copy-to-clipboard": "^3.2.0",
```

Comment:
Do we really need an npm package to copy a string to the clipboard? can't we just use the browser API for this? https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard

Reply:
Nice, I didn't realize it was that straightforward now!

Reply:
I think @BoulderBrains ran into issues using the native approach that were solved with this plugin instead. No need to refactor that along with all the other changes that are happening here already.

Reply:
@mcstover @eddieferrer I was able to get it working in Chrome and Firefox (but not Safari) with just [a few lines](https://github.com/kiva/ui/pull/1332/files#diff-6836054dccee092bf758bc335cfd3254R87) and no package. For the purposes of copying color values in Storybook I think that should be good enough.